### ISSUE TITLE
Points?

### ISSUE THREAD

**Issue by Steve3586** [raithos/xwing#534]
Hi,
Why do all of your ship/pilots show 1 point more than the actual list value?

**Comment by Patschke** [comment]
For me the points seem to show correctly. Are you aware of the last point update? Maybe the ships you refere to just got more expensive? 

Could you be more specific, e.g. which ship/pilot is affected?

**Comment by Steve3586** [comment]
Hi,
I'm working from the wave 6 points update released and dated 29th & 30th
Jan 2020 as listed by ffg, I'm not aware of any changes post Jan 2020. The
first two I looked at were Maarek Stele, he's listed as 45 but when I add
him to the squad builder he's listed at 46. An academy pilot is listed as
22 and you have it at 23. At first glance it appears to be everything
hyperspace legal but the numbers show the same when entered for both
hyperspace and extended (including starting from scratch each time and not
expecting entries to change if mode is changed).
Am I missing something?

Thanks in advance,

Steve

**Comment by Patschke** [comment]
Both of them have gotten 1 point cheaper during the Jan 2020 update. Did you try reloading with clearing the cache (Ctrl + F5)? Maybe your browser is still using a cached old version? Are you by accident using the /old version? 

Both pilots work fine for me. 

![grafik](https://user-images.githubusercontent.com/17853362/74087073-5b3e3580-4a89-11ea-94d1-18e3a7a6c64d.png)
![grafik](https://user-images.githubusercontent.com/17853362/74087100-8fb1f180-4a89-11ea-9cfd-a5fa8011feba.png)

If none of those is the case, what device/browser are you using? Is this also the case for pilots that did not change?

**Comment by Steve3586** [comment]
Fixed, thank you. I've no idea how I was getting the old page as I only opened it for the first time yesterday, but it's working now. Stay safe and have a magical day! 
-------- Original message --------From: Patrick Mischke <notifications@github.com> Date: 08/02/2020  14:44  (GMT+00:00) To: raithos/xwing <xwing@noreply.github.com> Cc: Steve3586 <stevewkirtley@gmail.com>, Author <author@noreply.github.com> Subject: Re: [raithos/xwing] Points? (#534) Both of them have gotten 1 point cheaper during the Jan 2020 update. Did you try reloading with clearing the cache (Ctrl + F5)? Maybe your browser is still using a cached old version? Are you by accident using the /old version?
Both pilots work fine for me.


If none of those is the case, what device/browser are you using? Is this also the case for pilots that did not change?

—You are receiving this because you authored the thread.Reply to this email directly, view it on GitHub, or unsubscribe.
[
{
"@context": "http://schema.org",
"@type": "EmailMessage",
"potentialAction": {
"@type": "ViewAction",
"target": "https://github.com/raithos/xwing/issues/534?email_source=notifications\u0026email_token=AOBYTHG7CWDVB2NPCGCC4PTRB3ALHA5CNFSM4KRXTNGKYY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOELFTIVY#issuecomment-583742551",
"url": "https://github.com/raithos/xwing/issues/534?email_source=notifications\u0026email_token=AOBYTHG7CWDVB2NPCGCC4PTRB3ALHA5CNFSM4KRXTNGKYY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOELFTIVY#issuecomment-583742551",
"name": "View Issue"
},
"description": "View this Issue on GitHub",
"publisher": {
"@type": "Organization",
"name": "GitHub",
"url": "https://github.com"
}
}
]
